src/server/object_services/md_store.js

```diff
@@ -669,7 +669,7 @@ class MDStore {
                 $lt: new Date(moment.unix(max_create_time).toISOString()),
                 $exists: true
             } : undefined,
-            tagging: tagging ? {
+            tagging: (tagging && tagging.length) ? {
```

Comment:
maybe be more explicit:
```suggestion
            tagging: (tagging && tagging.length > 0) ? {
```

Reply:
sure, thanks